Simplify and yet improve the description (#80)
openSUSE/jeos-firstboot: README.md

# jeos-firstboot
## Description
jeos-firstboot allows initial configuration and adjustments of a Linux system using text based dialogs.

> It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration..

Reviewer: ```suggestion
It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. This includes showing the license and prompt for language, keyboard layout, timezone, root passsword and configuring the network.
```

> It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration..

Reviewer 2: ```suggestion
It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration.
```

> This was mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).

Reviewer 2: ```suggestion
This is mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).
```

> This was mainly developed to work with openSUSE JeOS and SUSE Linux Enterprise Server JeOS images, more information visit [JeOS wiki](https://en.opensuse.org/Portal:JeOS)

Reviewer: ```suggestion
This was mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).
```
